src/web/app/pages-instructor/instructor-page.component.ts

```diff
@@ -44,14 +44,15 @@ export class InstructorPageComponent implements OnInit {
       display: 'Help',
     },
   ];
-  isFetchingAuthDetails: boolean = true;
+  isFetchingAuthDetails: boolean = false;
 
   private backendUrl: string = environment.backendUrl;
 
   constructor(private route: ActivatedRoute, private authService: AuthService) {}
 
   ngOnInit(): void {
     this.route.queryParams.subscribe((queryParams: any) => {
+      this.isFetchingAuthDetails = true;
```

Comment:
This should be set outside the `this.route.queryParams.subscribe` block? Same for the others where applicable.